nginx.conf

```diff
@@ -466,7 +466,9 @@ http {
     rewrite ^/comply/rooms/?$ https://www.conveyorhq.com/customer-trust-management/rooms permanent;
     rewrite ^/blog/common_HIPAA_questions.html$ /hipaa/ permanent;
     rewrite ^/company/owners-manual$ /owners-manual/ permanent;
-    rewrite ^/support/toolbelt$ https://deploy-docs.aptible.com/docs/cli permanent;
+    rewrite ^/support/toolbelt/$ https://deploy-docs.aptible.com/docs/cli permanent;
```

Comment:
@almathew 

I'm trying to fix https://aptible.com/support/toolbelt/  (which is currently a live backlink, but 404's).

We also want https://aptible.com/support/toolbelt to continue working.  

Is this change the correct rule or do I need to rules for both of these to work?

Reply:
Sorry, I messed this up. It looked convincingly right at the time for some reason, but the correct value is `/?$`. I'm fixing in another PR